packages/zoe/tools/fakePriceAuthority.js

```diff
@@ -50,9 +85,13 @@ export function makeFakePriceAuthority(amountMaths, priceList, timer) {
     });
   }
 
-  function priceOutQuote(currentTime, brandIn, amountOut) {
-    const mathIn = amountMaths.get(brandIn.getAllegedName());
-    const mathOut = amountMaths.get(amountOut.brand.getAllegedName());
+  /**
+   * @param {Brand} brandIn
+   * @param {Amount} amountOut
+   * @param {Timestamp} currentTime
+   */
+  function priceOutQuote(brandIn, amountOut, currentTime) {
```

Comment:
Thanks for re-ordering the params. I meant to do that.  

Would you mind changing `currentTime` to `quoteTime`?  Both `price*Quote()` methods started out with it being `currentTime`, and being passed into an earlier version of `currentPrice()`. Now `currentPrice()` knows the time, and the parameter is only used to label the quote.  When a `wake()` is late, the time given is the intended time, which is what should go on the quote.